lib/utils/indent-common.js

```diff
@@ -821,6 +850,8 @@ module.exports.defineVisitor = function create (context, tokenStore, defaultOpti
       if (node.name !== 'pre') {
         const childTokens = node.children.map(n => tokenStore.getFirstToken(n))
         setOffset(childTokens, 1, startTagToken)
+      } else {
+        setPreformattedTokens(node)
```

Comment:
My memory is uncertain, but I'd like to try a suggestion.
There is `ignore(node)` function to ignore tokens in a given node. Can we use it here?

Reply:
`ignore(node)` ignores the indentation of Token, but it can not ignore line indentation.
So, I think that it can not be used to ignore indentation of `<pre>` contents.

Reply:
How about this handler?

```js
    // Ignore inside of <pre>.
    "VElement[name='pre']:exit" (node) {
      const { endTag, children } = node
      for (const child of children) {
        ignore(child)
      }
      if (endTag) {
        ignore(endTag)
      }
    },
```

Reply:
I got it. Thank you for the confirming!